pkg/gateways/http/accounts/create.go

```diff
@@ -28,12 +28,24 @@ type CreateResponse struct {
 
 func (h Handler) Create(w http.ResponseWriter, r *http.Request) {
 
+	log := h.logger
+	requestID := r.Header.Get("Request-Id")
+	if requestID == "" {
+		log.Error("no request id informed")
+		w.Header().Add("Content-Type", server_http.JSONContentType)
+		response := server_http.Error{Reason: "invalid request header"}
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(response)
+		return
+	}
+	log = log.WithField("request_id", requestID)
+
 	var createRequest CreateRequest
 	err := json.NewDecoder(r.Body).Decode(&createRequest)
 	if err != nil {
 		w.Header().Add("Content-Type", server_http.JSONContentType)
 		response := server_http.Error{Reason: "invalid request body"}
-		log.Printf("error decoding body: %s\n", err)
+		log.WithError(err).Errorf("error decoding body: %s\n", err)
```

Comment:
```suggestion
		log.WithError(err).Errorf("error decoding body: %s", err)
```
there's no need of new line =)